src/umocktypes_wcharptr.c

```diff
@@ -1,13 +1,13 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-#include <errno.h>
 #include <string.h>
 #include <stddef.h>
 #include <stdlib.h>
 #include <wchar.h>
+#include <errno.h> // IWYU pragma: keep
```

Comment:
>keep [](start = 35, length = 4)

Is there a way to put this in the config instead? Like make errno a symbol:
`{ symbol: ["errno", "private", "<errno.h>", "public"] },`
Just checking

Reply:
This is an oddness where iwyu thinks errno is not needed, but actually it is. The only way to fix it is to inform iwyu that it is wrong :-). By a pragma. The rules mapping file that iwyu supports does not allow for expressing a "keep".